canton/community/ledger/ledger-api-core/src/main/scala/com/digitalasset/canton/platform/apiserver/meteringreport/MeteringReport.scala

```diff
@@ -35,7 +35,7 @@ object MeteringReport {
   final case class ApplicationReport(application: ApplicationId, events: Long)
 
   implicit val TimestampFormat: RootJsonFormat[Timestamp] =
-    stringJsonFormat(Timestamp.fromString)(_.toString)
+    stringJsonFormat(Timestamp.fromString(_))(_.toString)
```

Comment:
cc @remyhaemmerle-da What is the process to get that into canton proper?

Reply:
team-sdk-update is taking care of that.
However, they need a snapshot to that, and we did not publish snapshot until this morning.